app/views/admin/order_cycles/_exchange_form.html.haml

```diff
@@ -26,7 +26,7 @@
 
         = select_tag 'order_cycle_{{ exchangeDirection(exchange) }}_exchange_{{ $parent.$index }}_enterprise_fees_{{ $index }}_enterprise_fee_id', nil, {'id' => 'order_cycle_{{ exchangeDirection(exchange) }}_exchange_{{ $parent.$index }}_enterprise_fees_{{ $index }}_enterprise_fee_id', 'ng-model' => 'enterprise_fee.id', 'ng-options' => 'enterprise_fee.id as enterprise_fee.name for enterprise_fee in enterpriseFeesForEnterprise(enterprise_fee.enterprise_id)'}
 
-        = link_to 'Remove', '#', {'id' => 'order_cycle_{{ exchangeDirection(exchange) }}_exchange_{{ $parent.$index }}_enterprise_fees_{{ $index }}_remove', 'ng-click' => 'removeExchangeFee($event, exchange, $index)'}
+        = link_to t('remove'), '#', {'id' => 'order_cycle_{{ exchangeDirection(exchange) }}_exchange_{{ $parent.$index }}_enterprise_fees_{{ $index }}_remove', 'ng-click' => 'removeExchangeFee($event, exchange, $index)'}
```

Comment:
should be t('.remove') to pickup the lazy lookup, otherwise it will lookup in the root of the translations keys.

Reply:
I think @m-budryte 's idea was to use the global "remove" key present in the root of the file. Not sure which is better though.

Reply:
I've done it .remove first but order_cycle.exchanges.remove was not defined. Then I've added the keys inside exchange in every language file by copying the line from the global 'remove'. Then I was told not to change the localisation files as it's done through a different system. Current solution fixes the lack of translation straightaway and does not require extra work from translators but not sure if it's good for the future. :)

Reply:
I am sorry but this PR, as is, is not correct.
you can either:
1. use the root remote t('remove') already [here](https://github.com/openfoodfoundation/openfoodnetwork/blob/b443f5ece9490539c6e1b55cdefe2291317b02ea/config/locales/en.yml#L208), no change to en.yml
2. use relative and lazy look up t('.remove') and add the new relative key in en.yml as you are going in this PR

I approve both but I prefer option 2.

Reply:
ok, I added one commit, we go with option 2.